PiwikTracker-Swift/CustomVariable.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  CustomVariable.swift
+//  PiwikTracker
+//
+//  Created by Cornelius Horstmann on 09.10.16.
+//  Copyright © 2016 Mattias Levin. All rights reserved.
+//
+
+import Foundation
+
+@objc public enum CustomVariableScope: Int {
```

Comment:
Could consider moving the enum inside the class below ans just rename it to `Scope`
(you will get some namespace help)